index.html

```diff
@@ -364,13 +364,13 @@
                     <table class="simple">
                         <thead><tr><th>Current state of device buttons</th><th><code>buttons</code></th></tr></thead>
                         <tbody>
-                            <tr><td><b>Mouse move with no buttons pressed</b></td><td>0</td></tr>
-                            <tr><td>Left Mouse,<br>Touch Contact,<br>Pen contact (with no modifier buttons pressed)</td><td>1</td></tr>
+                            <tr><td><b>Mouse moved with no buttons pressed</b><br>, Pen moved without changing its <a>active buttons state</a></td><td>0</td></tr>
```

Comment:
Unless I'm misunderstanding this, would this not mean that for a hovering pen, the first time the user presses one of the buttons, they'd get a non-zero `buttons`, but on subsequent move they'd get 0 regardless of what buttons are pressed as it's not "transitioning" per the new active buttons definition?